### DISCUSSION TITLE
Could not do a head request for...

### DISCUSSION THREAD

**Discussion by kekec777** [containrrr/watchtower#1571] [General] [answered]
So i'm hoping you guys can figure this one out..

I have 4 images , where i use dockerfile and one of them is wordpress , just to give an example.. 
Dockerfile:
```Dockerfile
FROM wordpress:latest
RUN chmod o+r /etc/resolv.conf && apt-get update
```

Here i use "docker-compose up -d wordpress" to pull & build and start the images , this goes fine..

So here's the issue.. 

When watchtower checks for newer images for wordpress , it fails:
```
Could not do a head request for "dockercompose-wordpress:latest", falling back to regular pull.
Reason: registry responded to head request with "401 Unauthorized", auth: "Bearer realm=\"https://auth.docker.io/token\",service=\"registry.docker.io\",scope=\"repository:library/dockercompose-wordpress:pull\",error=\"insufficient_scope\""
Unable to update container "/wordpress": Error response from daemon: pull access denied for dockercompose-wordpress, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

and for me it seems the name is wrong = dockercompose-wordpress.:latest
And thats a name my docker-compose is setting and i dont know how to change it or if i could would it solve the problem..

Any input is appreciated and thanks in advice :)

**Comment by piksel** [comment, marked as answer]
The prefix is what docker-compose adds, but that's not the issue here. Since you are creating your own image from a Dockerfile, there will never be any updates to it unless you build it yourself.
For watchtower to be able to check the container for updates it needs to be started from the image ref that you want to check for updates (in this case `wordpress:latest`) and not be built locally.

**Reply by kekec777** [reply]
> The prefix is what docker-compose adds, but that's not the issue here. Since you are creating your own image from a Dockerfile, there will never be any updates to it unless you build it yourself. For watchtower to be able to check the container for updates it needs to be started from the image ref that you want to check for updates (in this case `wordpress:latest`) and not be built locally.

Hmm so this is impossible? 
i mean , compose wordpress wih dockerfile and let watchtower update it / build it agian when there's a new image from wordpress?

**Reply by piksel** [reply]
Yeah, it's outside the scope of watchtower unfortunately.
